app/assets/javascripts/core/posts_controller.coffee

```diff
@@ -0,0 +1,50 @@
+#= require './view_controller'
+
+class Spaces.PostsController extends Spaces.ViewController
+
+	initialize: ->
+		($ 'body').on 'tagsUpdated', () ->
```

Comment:
coffeescript style point: don't have empty `()`, this line should look like `($ 'body').on 'tagsUpdated', ->`